packages/rest-crud/README.md

```diff
@@ -4,22 +4,21 @@ REST API controller implementing default CRUD semantics.
 
 ## Overview
 
-This module allows applications to quickly expose a model via REST API without
-having to implement a custom controller class.
+This module allows applications to quickly expose models via REST API without
+having to implement custom controller or repository classes.
 
 ## Installation
 
 ```sh
 npm install --save @loopback/rest-crud
 ```
 
-## Basic use
+## Creating a CRUD Controller
```

Comment:
Please preserve `Basic use` section, it's part of our README guidelines - see https://loopback.io/doc/en/contrib/README-guidelines.html

Proposed structure:

```
## Basic use

(Put a short paragraph here to quickly introduce the sections below.)

### Creating a CRUD Controller

(...)

### Creating a CRUD repository

(...)

### Integrated example

(...)
```